File: app/src/main/java/idrisadetunmbi/githubusers/services/GithubApi.java
package idrisadetunmbi.githubusers.services;

import idrisadetunmbi.githubusers.models.GithubUsersResponse;
import retrofit2.Call;
import retrofit2.http.GET;

public interface GithubApi {
    @GET("search/users?q=location:nairobi+language:java")
    Call<GithubUsersResponse> getUsers();
}

File: app/src/main/java/idrisadetunmbi/githubusers/services/GithubApi.kt
package idrisadetunmbi.githubusers.services

import idrisadetunmbi.githubusers.models.GithubUsersResponse
import retrofit2.Call
import retrofit2.http.GET

interface GithubApi {
    @get:GET("search/users?q=location:nairobi+language:java")
    val users: Call<GithubUsersResponse>
}
